res/css/views/elements/_AccessibleButton.scss

```diff
@@ -17,151 +17,151 @@ limitations under the License.
-            background-color: $button-danger-disabled-bg-color;
-        }
+    &.mx_AccessibleButton_disabled {
+        color: $button-danger-disabled-fg-color;
+        background-color: $button-danger-disabled-bg-color;
+    }
+}
 
-        .mx_AccessibleButton_kind_danger_outline {
-            color: $alert;
-            background-color: transparent;
-            border: 1px solid $alert;
+.mx_AccessibleButton_kind_danger_outline {
+    color: $alert;
+    background-color: transparent;
+    border: 1px solid $alert;
 
-            &.mx_AccessibleButton_disabled {
-                color: $button-danger-disabled-bg-color;
-                border-color: $button-danger-disabled-bg-color;
-            }
-        }
+    &.mx_AccessibleButton_disabled {
+        color: $button-danger-disabled-bg-color;
+        border-color: $button-danger-disabled-bg-color;
+    }
+}
 
-        .mx_AccessibleButton_kind_danger_sm {
-            &.mx_AccessibleButton_disabled {
-                color: $button-danger-disabled-fg-color;
-                background-color: $button-danger-disabled-bg-color;
-            }
-        }
+.mx_AccessibleButton_kind_danger_sm {
+    &.mx_AccessibleButton_disabled {
+        color: $button-danger-disabled-fg-color;
+        background-color: $button-danger-disabled-bg-color;
+    }
+}
 
-        .mx_AccessibleButton_kind_link {
-            color: $accent;
-            padding: 0;
-        }
+.mx_AccessibleButton_kind_link {
+    color: $accent;
+    padding: 0;
+}
 
-        .mx_AccessibleButton_kind_link_inline {
-            color: $accent;
-            font-size: inherit;
-            padding: 0 2px;
-        }
+.mx_AccessibleButton_kind_link_inline {
+    color: $accent;
+    font-size: inherit;
+    padding: 0 2px;
+}
 
-        .mx_AccessibleButton_kind_confirm_sm,
-        .mx_AccessibleButton_kind_cancel_sm {
-            padding: 0px;
-            width: 16px;
-            height: 16px;
-            border-radius: 100%;
-            position: relative;
-            display: block;
-
-            &::before {
-                content: "";
-                display: block;
-                position: absolute;
-                top: 0;
-                bottom: 0;
-                left: 0;
-                right: 0;
-                background-color: #ffffff;
-                mask-repeat: no-repeat;
-                mask-position: center;
-                mask-size: 80%;
-            }
-        }
+.mx_AccessibleButton_kind_confirm_sm,
+.mx_AccessibleButton_kind_cancel_sm {
+    padding: 0px;
+    width: 16px;
+    height: 16px;
+    border-radius: 100%;
+    position: relative;
+    display: block;
+
+    &::before {
+        content: "";
+        display: block;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        background-color: #ffffff;
+        mask-repeat: no-repeat;
+        mask-position: center;
+        mask-size: 80%;
```

Comment:
And everywhere for that matter, these classes should only be able to be used in conjunction IMO

Reply:
Yes, that totally makes sense. Please wait for a minute